docs/book/features/block_volume.md

```diff
@@ -0,0 +1,278 @@
+# Volume Provisioning
+
+There are two types of volume provisioning in a kubernetes cluster:
+
+1. Dynamic Volume Provisioning
```

Comment:
Can we also mention links to `volume_topology.md`, to let user know if they are interested in provisioning volume using topology, and use WaitForFirstConsumer volume binding mode with dynamic volume provisioning.

https://brave-kare-67c7d0.netlify.app/features/volume_topology.html

Note: volume topology is only supported for vanilla block volume driver only.